build/Publish/PublishNupkgToTransportFeed.targets

```diff
@@ -1,63 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Why don't we need this anymore? Repo toolset or something else is going to handle transport feed?

Reply:
I'll look into this.